### DISCUSSION TITLE
Blowfish error on windows?

### DISCUSSION THREAD

**Discussion by AJVenza** [nunocoracao/blowfish#1844] [Q&A]
Hello, I'm trying to install blowfish and have tried the different ways to do it but I seem to always get this error

error: invalid path 'exampleSite/content/users/Handbook on Teaching Empirical Software Engineering: Online Materials/feature.jpg'
fatal: unable to checkout submodule 'themes/blowfish'

Anyone know a fix?

**Comment by yannlemos** [comment]
Hey @AJVenza, seems to be windows-specific bug because there's forbidden characters in the printed folder. Bunch of people had this problem (including me) but it seems to have been fixed in [#1839](https://github.com/nunocoracao/blowfish/pull/1839). 

Guess it's just a matter of waiting a little bit until it's merged next release

**Comment by pranavnasrani** [comment]
Now fixed. Release v2.80.0 is out.
